src/components/Playbooks/Runs/Actions/PlaybookRunsApprovalActionItem.tsx

```diff
@@ -0,0 +1,33 @@
+import clsx from "clsx";
+import { FaCheckCircle } from "react-icons/fa";
+
+type PlaybookRunsActionItemProps = {
+  onClick?: () => void;
+  isSelected?: boolean;
+  stepNumber: number;
+};
+
+export default function PlaybookRunsApprovalActionItem({
+  onClick = () => {},
+  isSelected = false,
+  stepNumber
+}: PlaybookRunsActionItemProps) {
+  return (
+    <div
+      role="button"
+      className={clsx(
+        `flex flex-row items-center justify-between rounded border border-gray-200 px-4 py-2 hover:bg-gray-200`,
+        isSelected ? "bg-gray-200" : "bg-white"
+      )}
+      onClick={onClick}
+    >
+      <div className="flex flex-col">
+        <div className="flex flex-row items-center gap-2 text-sm text-gray-600">
+          <FaCheckCircle className="text-green-500" />
+          Approval
```

Comment:
Can we change this to the approval details, so you don't need to click on it to see who approved